### ISSUE TITLE
core 3.1.1 upload issues

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/bridge#214]
Package Versions
```
core --version: 3.1.1
```
```
node --version: 4.5.0
```
### Expected Behavior

clean exit

### Command

```

C:\Users\meije>storj -k meije0612 upload-file 57b04b21dd072de41858a79f H:\AnalogueModels\GS_B45\Top_View*

```

### Error

```

[Tue Sep 06 2016 23:00:59 GMT+0200 (W. Europe Daylight Time)] [info] 48 file(s) to upload.
[Tue Sep 06 2016 23:00:59 GMT+0200 (W. Europe Daylight Time)] [info] Generating encryption key...
[Tue Sep 06 2016 23:00:59 GMT+0200 (W. Europe Daylight Time)] [info] Encrypting file "H:/AnalogueModels/GS_B45/Top_View/brittle 45.JPG"
[Tue Sep 06 2016 23:00:59 GMT+0200 (W. Europe Daylight Time)] [info] [ brittle 45.JPG ] Encryption complete!
[Tue Sep 06 2016 23:00:59 GMT+0200 (W. Europe Daylight Time)] [info] [ brittle 45.JPG ] Creating storage token...
[Tue Sep 06 2016 23:02:59 GMT+0200 (W. Europe Daylight Time)]brittle 45.JPG,
502 Bad Gateway

502 Bad Gateway


nginx/1.10.0

[Tue Sep 06 2016 23:02:59 GMT+0200 (W. Europe Daylight Time)] [info] [ brittle 45.JPG ] Cleaning up...
[Tue Sep 06 2016 23:02:59 GMT+0200 (W. Europe Daylight Time)] [info] [ brittle 45.JPG ] Finished cleaning!

C:\Users\meije>

```

**Comment by MeijeSibbel** [comment]
command: storj -k meije0612 -r 10 -c 200 upload-file 57b05629905cfece30b430f0 H:\MOviesP\Terminator_Genisys\Terminator.Genisys.mp4

[Tue Sep 06 2016 22:57:22 GMT+0200 (W. Europe Daylight Time)] [info] Data channel opened, transferring shard...
[Tue Sep 06 2016 22:57:23 GMT+0200 (W. Europe Daylight Time)] [info] Shard transfer completed! 3 remaining...
[Tue Sep 06 2016 22:57:24 GMT+0200 (W. Europe Daylight Time)] [info] Contract negotiated with: {"protocol":"0.8.0","address":"136.62.72.151","port":36502,"nodeID":"6034299a587ae04782dce3eaf493161292919288","lastSeen":1473195439968}
[Tue Sep 06 2016 22:57:24 GMT+0200 (W. Europe Daylight Time)] [info] Data channel opened, transferring shard...
[Tue Sep 06 2016 22:57:38 GMT+0200 (W. Europe Daylight Time)] [info] Shard transfer completed! 2 remaining...
[Tue Sep 06 2016 22:58:07 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 12)
[Tue Sep 06 2016 22:58:22 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 13)
[Tue Sep 06 2016 22:58:38 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 14)
[Tue Sep 06 2016 23:00:37 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 15)
[Tue Sep 06 2016 23:00:52 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 16)
[Tue Sep 06 2016 23:02:52 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 17)
[Tue Sep 06 2016 23:03:09 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 18)
[Tue Sep 06 2016 23:03:32 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 19)
[Tue Sep 06 2016 23:04:07 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 20)
[Tue Sep 06 2016 23:06:07 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 21)
[Tue Sep 06 2016 23:06:22 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 22)
[Tue Sep 06 2016 23:06:38 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 23)
[Tue Sep 06 2016 23:06:53 GMT+0200 (W. Europe Daylight Time)] [info] Querying bridge for contract for 465204b1c2600701b9eefdfacb9d68ac1104a4de (retry: 24)
[Tue Sep 06 2016 23:07:09 GMT+0200 (W. Europe Daylight Time)] [warn] Failed to upload shard...
[Tue Sep 06 2016 23:07:09 GMT+0200 (W. Europe Daylight Time)] [warn] [ Terminator.Genisys.mp4 ] Error occurred. Triggering cleanup...
[Tue Sep 06 2016 23:07:09 GMT+0200 (W. Europe Daylight Time)] [info] [ Terminator.Genisys.mp4 ] Cleaning up...
[Tue Sep 06 2016 23:07:09 GMT+0200 (W. Europe Daylight Time)] [info] [ Terminator.Genisys.mp4 ] Finished cleaning!
[Tue Sep 06 2016 23:07:09 GMT+0200 (W. Europe Daylight Time)] [error] [ undefined ] A file has failed to upload: Error: No storage offers were received

C:\Users\meije>

**Comment by MeijeSibbel** [comment]
### Command
```
storj -k meije0612 upload-file 57af3a1e8f266a7d4e391055 D:\films/**/*
```


### Error

```
 [Sat Sep 10 2016 13:21:32 GMT+0200 (W. Europe Daylight Time)] [info]   44 file(s) to upload.
 [Sat Sep 10 2016 13:21:34 GMT+0200 (W. Europe Daylight Time)] [info]   Generating encryption key...
 [Sat Sep 10 2016 13:21:37 GMT+0200 (W. Europe Daylight Time)] [info]   Encrypting file "D:/films/Brooklyn (2015) DVDSCR AVC-AAC filoumouton
RIP.mp4"
 [Sat Sep 10 2016 13:22:18 GMT+0200 (W. Europe Daylight Time)] [info]   [ Brooklyn (2015) DVDSCR AVC-AAC filoumoutonRIP.mp4 ] Encryption com
plete!
 [Sat Sep 10 2016 13:22:18 GMT+0200 (W. Europe Daylight Time)] [info]   [ Brooklyn (2015) DVDSCR AVC-AAC filoumoutonRIP.mp4 ] Creating stora
ge token...
 [Sat Sep 10 2016 13:23:09 GMT+0200 (W. Europe Daylight Time)]Brooklyn (2015) DVDSCR AVC-AAC filoumoutonRIP.mp4,<html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.10.0</center>
</body>
</html>

 [Sat Sep 10 2016 13:23:09 GMT+0200 (W. Europe Daylight Time)] [info]   [ Brooklyn (2015) DVDSCR AVC-AAC filoumoutonRIP.mp4 ] Cleaning up...

 [Sat Sep 10 2016 13:23:12 GMT+0200 (W. Europe Daylight Time)] [info]   [ Brooklyn (2015) DVDSCR AVC-AAC filoumoutonRIP.mp4 ] Finished clean
ing!

C:\Users\Dani Marion>

```

**Comment by bookchin** [comment]
This should get fixed by #218
